tfjs-core/src/ops/browser.ts

```diff
@@ -359,6 +370,10 @@ export async function toPixels(
   }
 
   if (canvas != null) {
+    console.warn(
+        'tf.browser.toPixels is not efficient to draw tensor on canvas. ' +
+        'Please try tf.browser.draw instead.');
```

Comment:
Nice catch. Added a variable in the browser.ts file for the boolean, since the file already has a variable `hasToPixelsWarned`, which share the coding style.